src/core/Settings.js

```diff
@@ -998,14 +998,11 @@ import Events from './events/Events.js';
  * - Constants.TRACK_SWITCH_MODE_NEVER_REPLACE
  * Do not replace existing segments in the buffer
  *
- * @property {string} [selectionModeForInitialTrack="highestSelectionPriority"]
+ * @property {string} [selectionModeForInitialTrack="highestEfficiency"]
```

Comment:
Can we keep `highestSelectionPriority` as an option here. I would still like to give applications the chance to ignore `selectionPriority` if required. With the changes here and below we always use `selectionPriority` regardless of the configured mode

Reply:
Done - see the newly introduced `ignoreSelectionPriority` flag